dev-python/mongoengine/mongoengine-0.25.0.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} )
+
+inherit distutils-r1
+
+DESCRIPTION="A Python Object-Document-Mapper for working with MongoDB"
+HOMEPAGE="
+	https://github.com/MongoEngine/mongoengine
```

Comment:
Personal preference: use a trailing slash for GH URLs.

Also please add the PyPI homepage.